requirements/_requirements.txt

```diff
@@ -3,6 +3,8 @@ cython<=3.0.0
 fastapi<=0.85.1
 numpy<=1.25.2
 opencv-python<=4.8.0.76
```

Comment:
Is this line redundant with the below headless install?

Reply:
I'm not sure, but I've seen some problems with differing version of opencv vs opencv headless so I'd just rather be sure